frontend/src/views/producer/dashboard/edition-form/Editor.jsx

```diff
@@ -406,14 +412,15 @@ class Editor extends React.Component {
           <Row>
             <InputField label="Age group limit" lightMode horizontal>
               <TagPillGroup
+                //multiple //@TODO: toogle this once EventModal gets updated
                 highlightColor={this.internalData.eventDraft.themeColor}
-                onChange={this.onPillChange('ageGroupLimit')}
+                onChange={this.onAgeGroupChange}
                 value={this.internalData.eventDraft.ageGroupLimit}
                 tags={[
```

Comment:
Maybe you could add a `constants.js`where you define all these tags an import them where they are needed, instead of inlining them every time inside render.. that way they would be easy to find and maintain in the future...

Reply:
True. I can add a trello ticket for that. The repetition is in many places though, so it will take time